<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="home">
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QLabel" name="welcomeLabel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>190</y>
        <width>491</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Welcome to Super Groupinator!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="manageClassButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>260</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Manage Classes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createClassButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>300</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Class</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="manageClass">
     <attribute name="title">
      <string>Manage Classes</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Class List</string>
      </property>
     </widget>
     <widget class="QWidget" name="classWidget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>231</width>
        <height>301</height>
       </rect>
      </property>
      <widget class="QListWidget" name="classList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>201</width>
         <height>191</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="studentWidget" native="true">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>491</width>
        <height>381</height>
       </rect>
      </property>
      <widget class="QListWidget" name="studentList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>471</width>
         <height>351</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>490</y>
        <width>242</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="viewClassButton">
         <property name="text">
          <string>View Class</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveClassButton">
         <property name="text">
          <string>Save Class</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteClassButton">
         <property name="text">
          <string>Delete Class</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="addStudentButton">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>440</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Student</string>
      </property>
     </widget>
     <widget class="QPushButton" name="removeStudentButton">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>440</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete Student</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="newClass">
     <attribute name="title">
      <string>New Class</string>
     </attribute>
     <widget class="QPushButton" name="createClassButton_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>270</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Class</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>190</y>
        <width>213</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="classNameLabel">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Class Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="classNameInput">
         <property name="placeholderText">
          <string>Enter Class Name</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="uploadFileButton">
      <property name="geometry">
       <rect>
        <x>271</x>
        <y>231</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Upload file</string>
      </property>
     </widget>
     <widget class="QLabel" name="fileNameLabel">
      <property name="geometry">
       <rect>
        <x>352</x>
        <y>236</y>
        <width>251</width>
        <height>16</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>No file selected</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="createGroup">
     <attribute name="title">
      <string>Create Groups</string>
     </attribute>
     <widget class="QLabel" name="generatorLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>531</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Group Generator</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>70</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="numStudentsGroup">
         <property name="text">
          <string># Students per group</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinStudentsGroups">
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>6</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="studentWidget_2" native="true">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>100</y>
        <width>491</width>
        <height>381</height>
       </rect>
      </property>
      <widget class="QListWidget" name="studentList_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>471</width>
         <height>351</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="classWidget_2" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>231</width>
        <height>301</height>
       </rect>
      </property>
      <widget class="QListWidget" name="classList_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>201</width>
         <height>271</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="createGroupsButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>410</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Groups</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exportGroupsButton">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>490</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Export Groups</string>
      </property>
     </widget>
     <widget class="QComboBox" name="selectClassCombo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Select Class</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPHYS_150">
   <property name="text">
    <string>PHYS 150</string>
   </property>
  </action>
  <action name="actionCS_20">
   <property name="text">
    <string>CS 120</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
